src/project_file_walker.rs

```diff
@@ -94,10 +95,10 @@ pub(crate) fn get_project_file_walker_types(language: Option<SupportedLanguage>)
     let mut types_builder = TypesBuilder::new();
     types_builder.add_defaults();
     if let Some(language) = language {
-        types_builder.select(language.name_for_ignore_select);
+        types_builder.select(get_supported_language_name_for_ignore_select(language));
     } else {
-        for language in ALL_SUPPORTED_LANGUAGES.iter() {
-            types_builder.select(language.name_for_ignore_select);
+        for name_for_ignore_select in SUPPORTED_LANGUAGE_NAMES_FOR_IGNORE_SELECT.iter() {
```

Comment:
Sort of interesting that with the approach of "moving all of these other fields into their own separate fixed-size data structures" now this usage pattern becomes an iteration over "the data structure that just has that one relevant thing in it"